api/file.go

```diff
@@ -76,7 +76,7 @@ func uploadFile(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	resStruct, err := c.App.UploadFiles(c.TeamId, channelId, c.Session.UserId, m.File["files"], m.Value["client_ids"])
+	resStruct, err := c.App.UploadFiles(c.TeamId, channelId, c.Session.UserId, m.File["files"], m.Value["client_ids"], false)
```

Comment:
this refers to the APIv3 which we shouldn't be doing anything to it at this point as is deprecated

Reply:
I hard-coded the false so that the method `App.UploadFiles` had the proper number of attributes

Reply:
Otherwise I guess I could make that false optional... is that the better solution?

Reply:
Oh I'm sorry I didn't see this was the call to the app layer, that's totally fine then